src/catalog/database_catalog.cpp

```diff
@@ -1883,6 +1897,22 @@ void DatabaseCatalog::BootstrapProcContexts(const common::ManagedPointer<transac
                                                            execution::ast::Builtin::NpRunnersDummyReal, true);
   SetProcCtxPtr(txn, postgres::NP_RUNNERS_DUMMY_REAL_PRO_OID, func_context);
   txn->RegisterAbortAction([=]() { delete func_context; });
+
+  func_context = new execution::functions::FunctionContext(
+      "extractYear", type::TypeId::INTEGER, {type::TypeId::INTEGER}, execution::ast::Builtin::ExtractYear);
```

Comment:
Can you work with @dniu16 to add SQL tests for this function?

Reply:
Do we have support for extractYear at SQL level? 
SELECT EXTRACT(YEAR FROM TIMESTAMP '2001-02-16 20:38:40'); doesn't work. May have to add some additional code

Reply:
#1068